test/bdd_2d.jl

```diff
@@ -0,0 +1,34 @@
+using Sinograms: bdd_2d
+using Test: @test, @testset, @inferred
+using JLD: load
+
+@testset "bdd_2d" begin
+    deg = 1
+    geo = (DSD = 100000, DSO = 99700, pSize = 1, dSize = 0.5, nPix = 256, nDet = 1024,
+          theta = deg2rad.(0:deg:360-deg), isoX = 0, isoY = 0)
+    phantomImg = shepp_logan(geo.nPix, SheppLoganToft())
+    sinogramB = projection(phantomImg',geo)
+    imageB = backprojection(sinogramB,geo)
+
+    sinogramB_correct = load("sinomat.jld")["sinomat"]
+    imageB_correct = load("imagemat.jld")["imagemat"]
```

Comment:
```suggestion
#   sinogramB_correct = load("sinomat.jld")["sinomat"]
#   imageB_correct = load("imagemat.jld")["imagemat"]
```
A code repo should only have source code in it, no data files like .jld files.
It was great that you tested it this way, but please remove the .jld files from your PR.